### ISSUE TITLE
downloader for text

### ISSUE THREAD

**Issue by orangebutblue** [xates/tg-channel-watcher#7]
Hi, thanks for the script by the way.

I was wondering, if it was possible to have the forwarder-functionality, but for downloading the text into a file (like the downloader).

So, if somebody used one of the forwarder-patterns, it would download the message either into a separate text-file (new file for each new message) or an aggregated text-file (one file for all messages)

**Comment by xates** [comment]
Yeah, why not... I might work on that... probably next week if I have some time.
Or you can make a pr if you'd like
